### ISSUE TITLE
Rocket shouldn't spawn in survivor mode

### ISSUE THREAD

**Issue by brian70123475** [red-eclipse/world#64]
For obvious reasons. ;)

**Comment by Dziq84** [comment]
hmmm if this going to be discussion/suggestions, a pros/cons are welcome. not just "for obvious reasons" :-P if this is obvious, then why rocket it is still there ;-) ?
I only can find this a little game breaking having a rocket in survivor, since its very annoying to just die by that OP weapon and wait for another turn which mostly leads me to quit, I suggest changing rocket to something like in my vars (rocket is a like any other weapon and can be deadly in skilled hands) just for this gamemode

**Comment by lambda-11235** [comment]
Limiting the rocket could also be bad. Players may assume standard servers won't change the behaviour of weapons from game to game. I would find that very annoying. Maybe the rocket could be a little less OP in general.

**Comment by brian70123475** [comment]
> If this is obvious, then why rocket it is still there so many years?

In 1.6.0, rocket doesn't spawn in survivor mode. And that's a good thing, because right now "survival" mode is really "reach the rocket spawn first" mode.

> I suggest changing rocket to something like in my vars, just for this gamemode (rocket turns into mini-rockets, weapon which is more or like any other weapon, and can be deadly in more skilled hands, not just BOOM [...] )

Hmmm, sounds interesting. In other words, still making rocket a weapon that kills players very quickly, but requires a little more skill, instead of just aiming in their general direction?

> ...or removed in general. Now it's very unfair, because players don't spawn in the same spot.

Quite true. At this point, I'm just brainstorming, but maybe we could switch to a method where you only have access to rocket after you reach a goal (probably getting a certain number of kills). If you've had enough time to get N kills, then you've also had enough time to reach rocket spawn.

**Comment by brian70123475** [comment]
Or better yet, don't have a rocket spawn at all, but just have it pop up in the player weapon slots after they reach the goal.

**Comment by lambda-11235** [comment]
@Dziq84 What's even more annoying is when only one player knows where the rocket is.

**Comment by qreeves** [comment]
Last time this was discussed I believe we settled on leaving this decision up to the map author as the rocket is a very situational item/weapon. Some levels just don't suit having it at all, especially small maps, while others take a fair amount of skill to even obtain. I don't think anybody ever reviewed all the available maps for this though.

**Comment by Dziq84** [comment]
mhmm I only fear that this will be abused, I personally will dissable Rocket in Kargo, on which I working right now (for survival).  rocket is maybe hard to reach - but only for new players.  experienced ones have no problem with this

**Comment by SniperGoth** [comment]
Rocket in survival is just super unbalanced.
Get it and you win the match, or don't get it, and you're doomed.
But rocket in standart matches should stay, cause otherwise we won't have any super weapon, which is a flagship of every arena shooter out there.
And making it a killstreak, no, just no, otherwise the game will just become COD-like, encouraging camping for kills to get the rocket, to then get even more kills.

**Comment by brian70123475** [comment]
> Rocket is maybe hard to reach - but only for new players. Experienced [to intermediate] ones have no problem with this.

Agreed. I remember when I first started playing the game, getting the rocket in suspended or castle seemed nearly impossible. Now it's trivially easy.

**Comment by JeffBobbo** [comment]
Rocket shouldn't be in survivor. It's too powerful a weapon and doesn't fit the idea of survival.

I also agree with SniperGoth that rocket should not become a killstreak reward type thing. Rocket already had some gimmicky mechanics for it's respawn, but can be used to throw a game in your advantage (even if you're the little guy). Kill streak reward turns it into the FPS equivalent of "the rich get richer".

**Comment by Calinou** [comment]
What about spawning the rocket after a while (1 minute or so) in survivor/duel mode? As a bonus, this could make long rounds shorter.

**Comment by SniperGoth** [comment]
Would still be a terrible thing.
Get it, and boom, gg, everyone's dead.
To be honest, who thought that rocket in surv/duel was ever a good idea? :p

**Comment by qreeves** [comment]
> What about spawning the rocket after a while (1 minute or so) in survivor and duel modes? As a bonus, this could help shorten long rounds.

If a round is going too long, this could actually be a good thing to draw people out (kinda like grouping everyone toward a middle point in a royale game, and what I think @Calinou was really trying to express). The problem would be coming up with a reasonable time limit before it spawns, as it would vary depending on the map size in some cases.
